### ISSUE TITLE
Icon in Visual editor toolbar are out of place

### ISSUE THREAD

**Issue by gagarine** [jthingelstad/foreground#331]
### Setup

- Browsers and versions: Firefox 61
- MediaWiki version: 1.31.0
- PHP version: 7
- Foreground version: 2.1
- Visual Editor: 0.1.0 (13a585a) 00:14, 31 May 2018

### Issue

The visual editor toolbar works but the icone are not rightly placed
![screen shot 2018-08-01 at 17 17 46](https://user-images.githubusercontent.com/164074/43530933-fbfffd3c-95ae-11e8-93df-debdd852e7e2.png)

**Comment by Hutchy68** [comment]
See #285

 There is a final solution for this but it would take a lot of work. I would have to do with not relying on JQuery but using css for a before: element of  everything.

**Comment by Hutchy68** [comment]
Oops. Wrong answer...  they were all inline last time I checked.  Perhaps the visual editor team change some settings again.

**Comment by gagarine** [comment]
I fixed with (hackish):

```
.oo-ui-toolbar-bar .oo-ui-tool .oo-ui-tool-link {
	padding-top: 0 !important;
}
```

**Comment by kghbln** [comment]
> I fixed with (hackish):

Affirmative, after upgrading to MW 1.31.1 and Foreground 2.2.0 the displacement showed on the demo wiki, too. After [adding this to Common.css](https://foreground.wikiproject.net/w/index.php?title=MediaWiki:Common.css&diff=prev&oldid=622) the displacement goes away. 

@Hutchy68 Probably worth shipping this with Foreground.

**Comment by Hutchy68** [comment]
Fixed now with PR #383
